src/DOM/Event/CompositionEvent.purs

```diff
@@ -0,0 +1,21 @@
+module DOM.Event.CompositionEvent (
+    module T,
+    eventToCompositionEvent,
+    data_
+    ) where
+
+import Data.Maybe (Maybe(..))
+import DOM.Event.Types (CompositionEvent, Event)
+import DOM.Event.Types (CompositionEvent, compositionEventToEvent, readCompositionEvent) as T
+
+eventToCompositionEvent :: Event -> Maybe CompositionEvent
+eventToCompositionEvent = _eventToCompositionEvent Just Nothing
```

Comment:
The FFI could be avoided for these kind of functions by using `Data.Foreign.unsafeReadTagged` (after `toForeign`ing the `Event`).